pkg/githubsource/receive_adapter.go

```diff
@@ -30,6 +30,12 @@ import (
 	gh "gopkg.in/go-playground/webhooks.v3/github"
 )
 
+const (
+	GhHeaderEvent     = "X-GitHub-Event"
+	GhHeaderDelivery  = "X-GitHub-Delivery"
+	CeExtensionPrefix = "CE-"
```

Comment:
nit: Go style wants acronyms to be all caps e.g. `CEExtensionPrefix`.

Reply:
`s/all caps/consistent caps/`, if they lead a private variable then all lowercase is fine too.